includes/options/class-amp-options-menu.php

```diff
@@ -495,6 +506,33 @@ function updateFieldsetVisibility() {
 		<?php
 	}
 
+	/**
+	 * AMP Stories section renderer.
+	 *
+	 * @since 1.2
+	 */
+	public function render_amp_stories() {
+		?>
+		<p>
+			<label for="enable_amp_stories">
+				<input id="enable_amp_stories" type="checkbox" name="<?php echo esc_attr( AMP_Options_Manager::OPTION_NAME . '[enable_amp_stories]' ); ?>" <?php checked( AMP_Options_Manager::get_option( 'enable_amp_stories' ) ); ?>>
+				<?php esc_html_e( 'Enable experimental support for the AMP Story post type.', 'amp' ); ?>
+			</label>
+		</p>
+		<p class="description">
+			<?php
+			echo wp_kses_post(
+				sprintf(
+					/* translators: %s is the AMP Stories landing page */
+					__( 'AMP Stories is a new visual storytelling for the open web. AMP stories immerse your readers in fast-loading full-screen experiences. Easily create visual narratives, with engaging animations and tappable interactions. The AMP story format is free and part of the open web and are available for everyone to try on their websites. They can be shared and embedded across the web without being confined to a closed ecosystem or platform. Read more about <a href="%s">AMP Stories</a>.', 'amp' ),
+					esc_url( 'https://amp.dev/about/stories' )
```

Comment:
Making the text shorter would be nice. WDYT about something like this: "AMP Stories is a visual storytelling format for the open web, which immerse your readers in fast-loading, full-screen, and visually rich experiences. Stories can be a great addition to your overall content strategy. Read more about <a href="%s">AMP Stories</a>."

Reply:
We should also mention that this should be used in combination with WordPress 5.2 Beta or the current version of the Gutenberg plugin.

Reply:
I think we could just hide the setting entirely if `register_block_type` is not defined. In reality we're going to drop support for WP 4.9 anyway, so I don't think this is a big concern.

Reply:
I stand corrected. WordPress 5.2 or the latest Gutenberg is required.